cmd/operator-registry/main.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	cli "github.com/operator-framework/operator-registry/cmd/cli"
+)
+
+func main() {
+	rootCmd := &cobra.Command{
+		Use:   "operator-registry",
```

Comment:
let's start the bikeshed: 

`opreg`?

Reply:
I liked what @njhale said at scrum: `opm`

Reply:
+1

Reply:
`krab` 🦀

Reply:
> I liked what @njhale said at scrum: `opm`

What does opm stand for?

Reply:
`op`erator `m`atters :)

Reply:
I think it was `operator package manager`

What about  `opreg` for the server stuff (configmap, appregistry, registry-server today), `opm` for bundle / index stuff (initializer today)?

Reply:
It makes sense to me but didn't we want to combine all these into a single binary?